default.nix

```diff
@@ -112,18 +113,13 @@ let
         snapshot = pkgs.writeText "npm-snapshot"
           (builtins.toJSON (snapshotFromPackageLockJson actualPackageLock));
 
-        buildInputs = [
```

Comment:
are those really not used anywhere?

Reply:
`buildInputs` was only used to be combined and define `newBuildInputs`, which is then used below as the `buildInputs` of the derivation :)